src/components/overview/ImageGallery.jsx

```diff
@@ -26,7 +26,7 @@ const ImageGallery = (props) => {
             }
 
             return (
-              <span className={styles.galleryThumbnailPhotoContainer}>
+              <span className={styles.galleryThumbnailPhotoContainer} key={index}>
 
 
                 <img className={thumbnailStyle}
```

Comment:
Is this supposed to be `styles.thumbnailStyle`? Only reason I noticed is because it is the same mistake I made this morning that Devon found.